[2022-01-27 16:08:50,781] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: cgi_40.load_to_hdfs 2022-01-27T16:08:39.381668+00:00 [queued]>
[2022-01-27 16:08:50,789] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: cgi_40.load_to_hdfs 2022-01-27T16:08:39.381668+00:00 [queued]>
[2022-01-27 16:08:50,789] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-01-27 16:08:50,789] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-01-27 16:08:50,789] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-01-27 16:08:50,797] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): load_to_hdfs> on 2022-01-27T16:08:39.381668+00:00
[2022-01-27 16:08:50,804] {standard_task_runner.py:52} INFO - Started process 84 to run task
[2022-01-27 16:08:50,806] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cgi_40', 'load_to_hdfs', '2022-01-27T16:08:39.381668+00:00', '--job-id', '8', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/cgi40_dag.py', '--cfg-path', '/tmp/tmpe7i0ljys', '--error-file', '/tmp/tmpu58rdx9y']
[2022-01-27 16:08:50,807] {standard_task_runner.py:77} INFO - Job 8: Subtask load_to_hdfs
[2022-01-27 16:08:50,833] {logging_mixin.py:104} INFO - Running <TaskInstance: cgi_40.load_to_hdfs 2022-01-27T16:08:39.381668+00:00 [running]> on host ca77e06d0f72
[2022-01-27 16:08:50,861] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=oulespaul@gmail.com
AIRFLOW_CTX_DAG_OWNER=ETDA
AIRFLOW_CTX_DAG_ID=cgi_40
AIRFLOW_CTX_TASK_ID=load_to_hdfs
AIRFLOW_CTX_EXECUTION_DATE=2022-01-27T16:08:39.381668+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-27T16:08:39.381668+00:00
[2022-01-27 16:08:50,912] {logging_mixin.py:104} INFO - 'Stored!'
[2022-01-27 16:08:50,916] {logging_mixin.py:104} INFO - {'FileStatuses': {'FileStatus': [{'accessTime': 1643299730893,
                                  'blockSize': 134217728,
                                  'childrenNum': 0,
                                  'fileId': 90583,
                                  'group': 'cloudera',
                                  'length': 11,
                                  'modificationTime': 1643299730912,
                                  'owner': 'cloudera',
                                  'pathSuffix': 'CGI_4.0_2017_2019_20220127160850.txt',
                                  'permission': '755',
                                  'replication': 1,
                                  'storagePolicy': 0,
                                  'type': 'FILE'}]}}
[2022-01-27 16:08:50,917] {python.py:151} INFO - Done. Returned value was: None
[2022-01-27 16:08:50,923] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=cgi_40, task_id=load_to_hdfs, execution_date=20220127T160839, start_date=20220127T160850, end_date=20220127T160850
[2022-01-27 16:08:50,939] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-01-27 16:08:50,978] {local_task_job.py:151} INFO - Task exited with return code 0
